doc/spectrographs.rst

```diff
@@ -14,21 +14,28 @@ tips for customizing the PypeIt file.
 ======================  =========   =======================================
 PypeIt Name             Telescope   Instrument
 ======================  =========   =======================================
-shane_kast_blue         Lick 3m     Kast dual spectrometer; blue camera
-shane_kast_red          Lick 3m     Kast dual spectrometer; red camera
-shane_kast_red_ret      Lick 3m     Kast dual spectrometer; red reticon
+gemini_gmos             Gemini      GMOS spectrometer
+gemini_gnirs            Gemini      GNIRS spectrometer
 keck_lris_blue          Keck        :doc:`lris` spectrometer; blue camera
 keck_lris_red           Keck        :doc:`lris` spectrometer; red camera
-keck_lris_red_longonly  Keck        :doc:`lris` spectrometer; red camera windowed
+keck_mosfire            Keck        MOSFIRE spectrometer; limited gratings tested
+keck_nires              Keck        NIRES spectrometer
 keck_nirspec_low        Keck        NIRSPEC spectrometer; low-dispersion
 keck_deimos             Keck        DEIMOS spectrometer
-gemini_gnirs            Gemini      GNIRS spectrometer
-wht_isis_blue           WHT         ISIS spectrometer; blue camera?
+lbt_luci1               LBT         LUCI-I spectrometer
+lbt_luci2               LBT         LUCI-II spectrometer
+magellan_fire_long      Magellan    FIRE spectrometer; longslit mode
```

Comment:
magellan_fire_long is a child of magellan_fire now. Please change to
magellan_fire          Magellan FIRE spectrometer; Echelle mode
magellan_fire_long Magellan FIRE spectrometer; Longslit high throughput mode

Reply:
@feigewang can you please make this change.

Reply:
Done